fedramp_low_rev_4/fedramp_low_rev_4.sp

```diff
@@ -0,0 +1,24 @@
+locals {
+  fedramp_low_rev_4_common_tags = merge(local.aws_compliance_common_tags, {
+    fedramp_low_rev_4 = "true"
+    type              = "Benchmark"
+  })
+}
+
+benchmark "fedramp_low_rev_4" {
+  title         = "Federal Risk and Authorization Management Program (FedRAMP - Low)"
```

Comment:
```suggestion
  title         = "Federal Risk and Authorization Management Program (FedRAMP Low, Revision 4)"
```

I think we should include the revision as well, so users can differentiate once we add Revision 5 later